### PR TITLE
Command enhancement for livewire:make using Laravel Prompts

### PR DESCRIPTION
**Pull request by PhiloNL**
The first step towards implementing Laravel Prompts. If you run `php artisan livewire:make` without any arguments it will prompt for the various options.

image

### CODE REVIEW

File: src/Features/SupportConsoleCommands/Commands/MakeCommand.php [hunk 1]
```diff
@@ -131,6 +136,41 @@ public function isReservedClassName($name)
...
+                default: false
+            )
+        )
+        {
+            $input->setOption('inline', true);
+        }
+
+        if(
+            $testSuite = select(
+                label: 'Do you want to create an test?',
```
Comment (chillbram) [inline src/Features/SupportConsoleCommands/Commands/MakeCommand.php:157]: Typo: "an" should be "a". Maybe it's even better to extend it to say `Do you want to create a test file?`, to clarify its functionality a little bit more

Comment (PhiloNL) [reply src/Features/SupportConsoleCommands/Commands/MakeCommand.php]: Thanks! Updated :)

**Comment by utsavsomaiya** [comment]
I think this code shouldn't be maintainable because there is a multiple scenario are covered!!

1. For first livewire installed user to display the livewire logo and open the github repo...
2. Other we need to use the stub files and validate it and then we create it!!

---
So can we create as laravel does? e.g. : https://github.com/laravel/installer/blob/master/src/NewCommand.php

**Comment by PhiloNL** [comment]
> I think this code shouldn't be maintainable because there is multiple scenario are covered!!
> 
> 
> 
> 1. For first livewire installed user to display the livewire logo and open the github repo...
> 
> 2. Other we need to use the stub files and validate it and then we create it!!
> 
> 
> 
> ---
> 
> So can we create as laravel does? e.g. : https://github.com/laravel/installer/blob/master/src/NewCommand.php
> 
> 

Sorry I don't follow. This PR is just an enhancement that will prompt for input if you don't provide any arguments instead of throwing an error.

**Comment by calebporzio** [comment]
Awesome, thanks Philo!

**Comment by calebporzio** [comment]
Hey @PhiloNL,I had to revert this, this is what showed up for me locally:

I tried tweaking it and couldn't get it. If you get the time, can you fix and re-PR - thanks!

image

**Pull request merged**